java/src/org/openqa/selenium/manager/SeleniumManager.java

```diff
@@ -18,6 +18,7 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.io.CharStreams;
+import com.google.gson.GsonBuilder;
```

Comment:
I believe we can use this class implemented by Simon to handle JSON payloads. 

https://github.com/SeleniumHQ/selenium/blob/572434ee4988ba64d57c0fc6bb491c2a8224747d/java/src/org/openqa/selenium/json/Json.java#L30

Reply:
I have just committed a second patch to this PR to use that internal JSON parser instead of GSON. I tested locally and it works in the same way than GSON.